x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -7213,33 +7213,27 @@
     "xpack.apm.tutorial.dotNetClient.download.textPre": "将代理软件包从 [NuGet]({allNuGetPackagesLink}) 添加到 .NET 应用程序。有多个       NuGet 软件包可用于不同的用例。\n\n对于具有 Entity Framework Core 的 ASP.NET Core       应用程序，请下载 [Elastic.Apm.NetCoreAll]({netCoreAllApmPackageLink}) 软件包。此软件包将自动将每个       代理组件添加到您的应用程序。\n\n 如果您希望最大程度减少依存关系，您可以将       [Elastic.Apm.AspNetCore]({aspNetCorePackageLink}) 软件包仅用于       ASP.NET Core 监测，或将 [Elastic.Apm.EfCore]({efCorePackageLink}) 软件包仅用于 Entity Framework Core 监测。\n\n 如果       仅希望将公共代理 API 用于手动检测，请使用 [Elastic.Apm]({elasticApmPackageLink}) 软件包。",
     "xpack.apm.tutorial.downloadServerRpm": "寻找 32 位软件包？请参阅[下载页面]({downloadPageLink})。",
     "xpack.apm.tutorial.downloadServerTitle": "寻找 32 位软件包？请参阅[下载页面]({downloadPageLink})。",
     "xpack.apm.tutorial.elasticCloud.textPre": "要启用 APM Server，请前往 [Elastic Cloud 控制台](https://cloud.elastic.co/deployments/{deploymentId}/edit)，并通过单击“添加容量”在部署编辑页面启用 APM 和 Fleet，然后单击“保存”。启用后，请刷新此页面。",
-    "xpack.apm.tutorial.flaskClient.configure.commands.setCustomApmServerUrlComment": "设置定制 APM Server URL（默认值：{defaultApmServerUrl}）",
     "xpack.apm.tutorial.flaskClient.configure.textPost": "有关高级用法，请参阅[文档]({documentationLink})。",
-    "xpack.apm.tutorial.goClient.configure.commands.setCustomApmServerUrlComment": "设置定制 APM Server URL（默认值：{defaultApmServerUrl}）",
     "xpack.apm.tutorial.goClient.configure.textPost": "有关高级配置，请参阅[文档]({documentationLink})。",
     "xpack.apm.tutorial.goClient.instrument.textPost": "有关检测 Go 源代码的详细指南，请参阅[文档]({documentationLink})。",
     "xpack.apm.tutorial.javaClient.download.textPre": "从 [Maven Central]({mavenCentralLink}) 下载代理 jar。**切勿**将该代理添加为您的应用程序的依赖项。",
     "xpack.apm.tutorial.javaClient.startApplication.textPost": "有关配置选项和高级用法，请参阅[文档]({documentationLink})。",
     "xpack.apm.tutorial.javaClient.startApplication.textPre": "添加 `-javaagent` 标志并使用系统属性配置代理。\n\n * 设置所需的服务名称（允许使用的字符：a-z、A-Z、0-9、-、_ 和空格）\n * 设置定制 APM Server URL（默认值：{customApmServerUrl}）\n * 设置 APM Server 机密令牌\n * 设置服务环境\n * 设置您的应用程序的基础软件包",
     "xpack.apm.tutorial.jsClient.enableRealUserMonitoring.textPre": "APM Server 默认禁用 RUM 支持。请参阅[文档]({documentationLink})以详细了解如何启用 RUM 支持。将 APM 集成用于 Fleet 时，会自动启用 RUM 支持。",
-    "xpack.apm.tutorial.jsClient.installDependency.commands.setCustomApmServerUrlComment": "设置定制 APM Server URL（默认值：{defaultApmServerUrl}）",
     "xpack.apm.tutorial.jsClient.installDependency.textPost": "框架集成（如 React 或 Angular）具有定制依赖项。请参阅[集成文档]({docLink})以了解详情。",
-    "xpack.apm.tutorial.nodeClient.configure.commands.setCustomApmServerUrlComment": "设置定制 APM Server URL（默认值：{defaultApmServerUrl}）",
-    "xpack.apm.tutorial.nodeClient.configure.textPost": "请参阅[文档]({documentationLink})以了解高级用法，包括如何使用 [Babel/ES 模块]({babelEsModulesLink})。",
+    "xpack.apm.tutorial.nodeClient.configure.textPost": "请参阅[文档]({documentationLink})以了解高级用法，包括如何用于 [Babel/ES 模块]({babelEsModulesLink})。",
```

Comment:
is this new translation intentional? was it just a change in i18n id?

Reply:
It looks there's a slight change in some of the characters in the translation if you look carefully. Just making sure this is intentional.
![Screenshot 2023-03-29 at 10 10 20 PM](https://user-images.githubusercontent.com/1967266/228710351-d07e4495-1d91-42b1-9336-fb3f10e42279.png)

Reply:
Huh. Indeed. I'll remove that. It wasn't intentional. There was a conflict that I resolved and this must've slipped in. Good find.